example_injector.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import win32con
+from injector import Injector
+from pywinauto import Desktop
+from ctypes import wintypes
+import win32con
+import ctypes
+
+msg_id_to_key = {getattr(win32con, attr_name): attr_name for attr_name in dir(win32con) if attr_name.startswith('WM_')}
+
+def print_winmsg(msg):
+    print("hWnd:{}".format(str(msg.hWnd)))
+    print("message:{}".format((msg_id_to_key[msg.message] if msg.message in msg_id_to_key else str(msg.message))))
+    print("wParam:{}".format(str(msg.wParam)))
+    print("lParam:{}".format(str(msg.lParam)))
+    print("time:{}".format(str(msg.time)))
+    print("pt:{}".format(str(msg.pt.x) + ',' + str(msg.pt.x)))
+
+if (len(sys.argv) != 2):
+    print("Usage: {} <Process name>".format(sys.argv[0]))
+    sys.exit(0)
+
+inj = Injector(Desktop(backend="win32")[sys.argv[1]])
```

Comment:
Not sure it will work. This is a window specification passed to `Injector` constructor, not an app. But maybe it's only naming issue?

Reply:
in injector it used only for get application handle because it works with WindowSpec object, i can rename app to sth more understandable